### ISSUE TITLE
Cannot get trace to be sent without a 'finish' annotation like Annotation.ClientRecv()?

### ISSUE THREAD

**Issue by jeffthompson1971** [openzipkin/zipkin-js#447]
I'm trying to figure out how to log a trace that simply tracks something like a 'page load' from the client.   I don't have a concept of 'start' and 'finish'.  I'm not trying to capture some 'duration' or time between a ClientSnd and ClientRecv.  I simply want to log that something happened.  But with zipkin-js, if I don't do something like:
 
tracer.recordAnnotation(new Annotation.**ClientRecv**()); 

**Then the trace just never gets sent.**   I how can i trigger zipkin to just send a trace with just some tags of information?   i did some testing around this and i find that the only way to get a trace to be sent is to 'finish' something like ProducerStop or ConsumerStop or ClientRcv.   I understand why we want to 'close off' the trace IFF indeed i'm trying to record the duration between a 'start' and 'stop' annotation.   But i don't have that with what i'm trying to do in the client.  I just want to log a trace for a 'page load' with the route so I can profile how users are using my application.

So in short my question is, why am I forced to use a ClientRecv or ConsumerStop or ServerRcv annotation to get a trace to be sent?   If there is a way to let zipkin know 'hey i'm done with this trace just send it' then please tell me what that is.

**Comment by jcchavezs** [comment]
Hi @jeffthompson1971 thanks for coming by, I suggest that for this kind of questions you come to `https://gitter.im/openzipkin/zipkin`. Regarding this issue you don't need the annotations, you can just do [local tracing](https://github.com/openzipkin/zipkin-js/tree/master/packages/zipkin#local-tracing). Please follow up on gitter.
